language-hask-source/language-hask-source.cabal

```diff
@@ -0,0 +1,46 @@
+cabal-version:       2.4
+
+name:                language-hask-source
+version:             0.1.0.0
+synopsis:            parser and AST for source language of Hask
+description:         See README for more information.
+homepage:            https://github.com/TheMatten/hask
+bug-reports:         https://github.com/TheMatten/hask/issues
+license:             GPL-3.0-only
+license-file:        LICENSE
+-- author: TODO
+-- maintainer: TODO
+-- copyright: TODO
```

Comment:
I'm not entirely sure how to fill this in in case of multiple maintainers.

Reply:
I've used "Hask maintainers" at the end - let me know if it should be something else.